docs/source/administrator/logging.md

```diff
@@ -0,0 +1,182 @@
+## Audit Trails
```

Comment:
Love these docs, it's great to have a detailed explanation and example of the newly introduced changes! 👍